packages/components/src/box-control/README.md

```diff
@@ -79,6 +79,22 @@ A callback function when an input value changes.
  - Required: No
  - Default: `() => {}`
 
+### `presets`
+
+Available presets to pick from.
+
+ - Type: `Preset[]`
+ - Required: No
+
+### `presetKey`
+
+The key of the preset to apply.
+If you provide a list of presets, you must provide a preset key to use.
```

Comment:
Potential improvements for Developer Experience:

- a runtime check, warning when only one of the two `presets` and `presetKey` props are defined
- conditional types, specifying that `presetKey` is not optional if the `presets` prop is specified